evo_researcher/benchmark/utils.py

```diff
@@ -20,12 +20,18 @@ class Market(BaseModel):
 
 
 class Prediction(BaseModel):
+    decision: t.Literal["y", "n"]
+    decision_token_prob: float
```

Comment:
Concerned about these being required fields, and also `_compute_mse_token_prob` being in the `predefined_metric_fns` map of the benchmarker, as this makes it a less generic tool - one that can't be used to benchmark agents that don't produce a token probability in their prediction

Reply:
Yes agreed, very good point -- I didn't realise.

I removed the token prob from the benchmark stuff but kept it in our agent's code. What I would do is create multiple agents as the input to the benchmark , some agent will use completion probability, some will use token probability and that way, it will be also nicer comparison in the benchmark report.

Wdyt?